gradio/components/audio.py

```diff
@@ -109,6 +109,7 @@ def __init__(
         max_length: int | None = None,
         waveform_options: WaveformOptions | dict | None = None,
         loop: bool = False,
+        stream_frequency: float = 2,
```

Comment:
Nitty nit, but we could call this `stream_every` to make it consistent with our other `every` parameter

Reply:
Could we consolidate this and the `streaming` arguments?

Reply:
Actually could we make this a parameter of .stream()? That way (a) different events could have different frequencies (b) this parameter and the time_limit() parameter are in the same place (c) if someone isn’t using streaming, they don’t have to think about this parameter

Reply:
Done

Reply:
> Actually could we make this a parameter of .stream()? That way (a) different events could have different frequencies (b) this parameter and the time_limit() parameter are in the same place (c) if someone isn’t using streaming, they don’t have to think about this parameter

Thoughts on this @freddyaboulton?

Reply:
Yes it's something I thought about but it's a bit awkward since we don't have a way of passing dependency props into components but regardless I will implement.